lib/console.js

```diff
@@ -234,6 +234,19 @@ Console.prototype.time = function time(label = 'default') {
   this._times.set(label, process.hrtime());
 };
 
+Console.prototype.timeLog = function timeLog(label = 'default', ...data) {
```

Comment:
Other than the function name printed in the process warning, couldn't `Console.prototype.timeEnd()` be updated to call this?